client/index.js

```diff
@@ -45,12 +51,14 @@ class Client {
     events.addEventListener('message', this.onmessage.bind(this))
     events.addEventListener('open', this.onopen.bind(this))
     events.addEventListener('error', this.onerror.bind(this))
-
-    this.logger.info(`Forwarding ${this.source} to ${this.target}`)
-
-    this.events = events
-
-    return events
+    if (this.validateURL(this.source) === true) {
+      this.logger.info(`Forwarding ${this.source} to ${this.target}`)
+      this.events = events
+
+      return events
+    } else {
+      this.logger.error('Please check your WEBHOOK_PROXY_URL in .env file')
```

Comment:
Smee won't always be run by Probot, so not everyone that uses it will pass the URL as this specific environment variable. Please change this to something like:

```
The provided URL is invalid
```

Reply:
OK.